swiftgen-cli/ArgumentsUtils.swift

```diff
@@ -62,6 +70,15 @@ enum OutputDestination: ArgumentConvertible {
   }
 }
 
+// MARK: Change comparsion
+
+extension Path {
+  func isContentEqualTo(string: String) throws -> Bool {
```

Comment:
I actually think this is fine. Using Objective-C the method name would probably be named `isContentEqualToString:`. As the `String` part is obvious in Swift, i dropped that part. 

Maybe dropping `is` from the name would make it even Swift-ier: `contentEqualTo:`.